services/cloudformation/src/it/java/software/amazon/awssdk/services/cloudformation/SendEmptyListIntegrationTest.java

```diff
@@ -83,9 +83,6 @@ public void setup() {
     @After
     public void tearDown() {
         cf.deleteStack(DeleteStackRequest.builder().stackName(stackName).build());
-        cf.waiters()
```

Comment:
Okay gotcha, I know for Dynamo there is a concurrent modification limit so it certain cases we do want to wait. But if the tests are consistently passing without this then I'd say leave if off.